app/src/main/java/com/sdp/movemeet/Navigation/Navigation.java

```diff
@@ -45,4 +46,10 @@ public static void goToChat(View view) {
         mContext.startActivity(intent);
     }
 
+    public static void goToListOfActivities(View view) {
+        Context mContext = view.getContext();
+        Intent intent = new Intent(mContext, ListActivity.class);
+        mContext.startActivity(intent);
```

Comment:
Didn't we say on Tuesday that our agreed convention was **NOT** to use the "m" prefix?

Reply:
My bad, I am sorry